docs/core-migration-guide.md

```diff
@@ -24,6 +24,8 @@ license: |
 
 ## Upgrading from Core 3.1 to 3.2
 
+- Since Spark 3.2, `spark.eventLog.compression.codec` has `zstd` by default which means Spark will not fallback to use `spark.io.compression.codec`. To restore the behavior before Spark 3.2, you can set `spark.eventLog.compression.codec` explicitly to the value of `spark.io.compression.codec`.
```

Comment:
nit but maybe: "`spark.eventLog.compression.codec` is set to `zstd` by default which means Spark will not fallback to use `spark.io.compression.codec` anymore"

Reply:
Sure.